src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/GetTaskByIdControllerTest.java

```diff
@@ -93,4 +93,83 @@ public void should_return_a_200_with_task_and_correct_properties() {
         common.cleanUpTask(taskId);
     }
 
+
```

Comment:
What about the other search endpoints?

1. SearchForCriteria
2. SearchForCompleteable

Should't we add the same tests?

Reply:
We have another ticket for search endpoint ```https://tools.hmcts.net/jira/browse/RWA-1376```